### PR TITLE
Fixes gadpathurian ship access issues + general tweaks

### PR DESCRIPTION
**Pull request by meep109**
I also added a cryo pod console and an additional power loader to the francisca gun.

### CODE REVIEW

**Comment by meep109** [comment]
!review

**Comment by meep109** [comment]
!bugfix

**Review by Generalcamo — APPROVED** [review]
I had originally intentionally removed the power loader for balance concerns. In hindsight however, making things inconvenient isn't a good way to balance things, so I approve of these changes. 

Thank you for this.

File: maps/away/ships/coc/gadpathur_patrol/gadpathur_patrol.dmm [hunk 1]
```diff
@@ -5449,7 +5578,11 @@
 /obj/machinery/atmospherics/pipe/manifold/hidden/fuel{
 	dir = 4
 	},
-/obj/machinery/alarm/east,
+/obj/machinery/alarm/west{
+	req_one_access = ssssss;
```
Comment (Geevies) [inline maps/away/ships/coc/gadpathur_patrol/gadpathur_patrol.dmm:5582]:
```suggestion
	req_one_access = list(223,224);
```

**Review by Geevies — CHANGES_REQUESTED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**